.github/workflows/release.yml

```
@@ -0,0 +1,48 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

```suggestion
      - 'v[0-9]+\.[0-9]+\.[0-9]+'
```
I suggest tagging with "v" as shown, but I am open to your preferences. Please feel free to merge with this tag if it aligns with your approach.

---

I prefer without a v prefix, but it's not my project. Whatever folks prefer here. Please note that the patterns are [glob-style](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) rather than regex patterns. So the dot should not be escaped.

---

we've been using the v in all of our version tags for [pyosmeta](https://github.com/pyOpenSci/pyosMeta/tags) so let's do the same here for consistency.

---

I suggest a different workflow here - instead of using tags - please deploy to PyPi on a release like we do with [pyosMeta](https://github.com/pyOpenSci/pyosMeta/blob/main/.github/workflows/publish-pypi.yml#L3).

This avoids the issue of having to create a tag with a specific string type and having regex in the build. And because we use zenodo for citation when a release is made a new zenodo doi is also create. 

Only a few of us will be making releases, which will simplify the build frequency.